build/common.mk

```diff
@@ -16,6 +16,9 @@ unit-test: lint
 integration-test: lint
 	go test -v -tags=integration github.com/basho/riak-go-client/...
 
+integration-test-hll: lint
+	go test -v -tags=integration_hll github.com/basho/riak-go-client/...
```

Comment:
I'll add this to `make.ps1`, since I'm the only person who cares about PS support :smile: